.buildkite/test-pipeline.yaml

```diff
@@ -7,8 +7,36 @@
 
 
 steps:
+- label: Async Engine, Inputs, Utils, Worker test
+  fast_check: true
+  fast_check_only: true
+  commands:
+  - pytest -v -s async_engine # Async Engine
+  - bash ../.buildkite/download-images.sh # Inputs 
+  - pytest -v -s test_inputs.py
+  - pytest -v -s multimodal
+  - pytest -v -s test_utils.py # Utils
+  - pytest -v -s worker # Worker
+
+- label: Tensorizer, Metrics, Tracings test
+  fast_check: true
+  fast_check_only: true
+  commands:
+  - apt-get install curl libsodium23 && pytest -v -s tensorizer_loader # Tensorizer
+  - pytest -v -s metrics # Metrics
+  - "pip install \
+      opentelemetry-sdk \
+      opentelemetry-api \
+      opentelemetry-exporter-otlp \
+      opentelemetry-semantic-conventions-ai" # Tracings
+  - pytest -v -s tracing
+  - cd /vllm-workspace/test_docs/docs
+  - pip install -r requirements-docs.txt
+  - SPHINXOPTS=\"-W\" make html
```

Comment:
```suggestion
- label: Tensorizer, Metrics, Tracing, Docs test
  fast_check: true
  fast_check_only: true
  commands:
  - apt-get install curl libsodium23 && pytest -v -s tensorizer_loader # Tensorizer
  - pytest -v -s metrics # Metrics
  - "pip install \
      opentelemetry-sdk \
      opentelemetry-api \
      opentelemetry-exporter-otlp \
      opentelemetry-semantic-conventions-ai" # Tracing
  - pytest -v -s tracing
  - cd /vllm-workspace/test_docs/docs # Docs
  - pip install -r requirements-docs.txt
  - SPHINXOPTS=\"-W\" make html
```

Reply:
Just a small nit.

Reply:
I guess that works too.

Reply:
fixed. Thanks!

Reply:
Actually let's split doc build to run individually since it's not running on GPU machine like other tests

Reply:
^ and it uses small CPU instance so cost shouldn't matter